openml/tasks/functions.py

```diff
@@ -275,31 +275,45 @@ def __list_tasks(api_call):
     return tasks
 
 
-def get_tasks(task_ids):
-    """Download tasks.
+def get_tasks(task_ids, download_data=True):
+    """Download task representation and data splits optionally.
     This function iterates :meth:`openml.tasks.get_task`.
     Parameters
     ----------
     task_ids : iterable
-        Integers representing task ids.
+        Integers/Strings representing task ids.
+    download_data : bool
+        Option to trigger download of data along with the meta data.
     Returns
     -------
     list
     """
     tasks = []
     for task_id in task_ids:
-        tasks.append(get_task(task_id))
+        tasks.append(get_task(task_id, download_data))
     return tasks
 
 
-def get_task(task_id):
-    """Download the OpenML task for a given task ID.
+def get_task(task_id, download_data=True):
+    """Download task representation for a given task ID
     Parameters
     ----------
-    task_id : int
+    The task representation is downloaded while the download of data
```

Comment:
Sorry, that's not how I meant it. Could you please check for example the doc strings of the [scikit-learn random forest](https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/ensemble/forest.py#L750)? There's a single line, then some additional information (where I think the sentence about the optional download of the split and data should go) and then the `Parameters`.

Reply:
sorry about that, should have looked up an example, will make the changes